test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 681000
+//   codeDepositCost: 680600
 //   executionCost: 715
-//   totalCost: 681715
+//   totalCost: 681315
```

Comment:
Interesting that in this case the optimized code is actually cheaper after this change.

Reply:
I'm actually not sure these changes are due to this PR...